### ISSUE TITLE
NVDA menu becomes unavailable after a time

### ISSUE THREAD

**Issue by nromey** [nvaccess/nvda#14518]
### Steps to reproduce:
Use NVDA over time, seems to have started occurring after Windows 11 22h2 and NVDA 2022.4 betas.
### Actual behavior:
NVDA menu, accessed by nvda+N does not read the submenu. See attached log for details on errors that seem to be generated. The NVDA menu is accessible using object nav, but invoking using nvda+enter does not work. To exit, you have to mouse click exit or restart NVDA using the NVDA hotkey.
### Expected behavior:
NVDA menu appears and does not become unavailable when using the NVDA+N key. I cannot get speech viewer information for this behavior.
### NVDA logs, crash dumps and other attachments:
[nvda-error.log](https://github.com/nvaccess/nvda/files/10365083/nvda-error.log)

#### NVDA installed/portable/running from source:
NVDA installed
#### NVDA version:
2022.4
#### Windows version:
Windows 11 22H2
#### Name and version of other software in use when reproducing the issue:
varies
#### Does the issue still occur after restarting your computer?
No, not right away, either restarting NVDA or restarting the computer brings back the menu. Te menu is diabled after a time however.
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Older versions did not exhibit this behavior
#### If NVDA add-ons are disabled, is your problem still occurring?
I disabled addons and received the same result.
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
yes

**Comment by nromey** [comment]
I downgraded to NVDA 2022.3.3 and there's been no issues.  I've noticed that right click seems to get stuck from time to time which is very random because I'm not touching the right mouse button. Once this happens, it appears as if it's a UIA issue, the menu becomes available. This should provide a more complete explanation of the issue.

**Comment by josephsl** [comment]
Hi,

We need a debug log please - althoug hte log provided does appear to suggest something going odd with UIA, it doesn't show exactly when and why that occurs, hence a debug log is needed to help us understand what's going on. If you don't mind, please upgrade to 2022.4 once again, restart NVDA with debug logging enabled, then upload the resulting debug log once the issue shows up again.

Thanks.

**Comment by nromey** [comment]
Thought I had debug enabled.  I had debug--warnining log enabled.
It's now enabled, NVDA is restarted and I'm awaiting the error.

**Comment by josephsl** [comment]
Hi, debug warning will print warnings, info, and error messages. The actual debug logging level will record key presses, speech/braille output, and other information specified in various NVDA modules and by add-ons. Thanks.

**Comment by nromey** [comment]
Debug log attached. [NVDA Error Log](https://www.dropbox.com/s/uyjqxq70ws087lh/nvda.log?dl=0)

**Comment by josephsl** [comment]
Hi, besides an issue with detecting a Vispero braille display and attempting to remove a nonexistent menu item from NVDA menu, does this happen after closing a specific app or a game (in this case, Hearthstone)? Thanks.

**Comment by nromey** [comment]
Hello,

Seems as if the issue may be hardware related.  I disabled my touchpad
as the system keeps saying that the right mouse button is getting
triggered, no matter what application I am in.  Since diabling the
touchpad, I haven't experienced the crash or the UIA issues--I can't
find in the log where the right mouse button is being activated, but
that seems to be what's happening.  I'm not sure why this would
trigger NVDA, but I can tell you that since disabling the touchpad,
the issue has stopped happening.

I'm not sure if this means we close the issue as regardless, I'm still
seeing the UIA issues when teh pad is enabled--I can get it replaced
and will, and this will probably get the thing to stop, but I wonder
why this triggers NVDA.  It's more of a curiosity.

**Comment by seanbudd** [comment]
Closing considering as this is a hardware issue.
